lighthouse-core/test/audits/first-meaningful-paint-test.js

```diff
@@ -39,7 +40,7 @@ describe('Performance: first-meaningful-paint audit', () => {
     const fmpResult = await FMPAudit.audit(artifacts, context);
 
     assert.equal(fmpResult.score, 0.79);
-    assert.equal(fmpResult.displayValue, '2,850\xa0ms');
+    assert.equal(Math.round(fmpResult.displayValue[1]), 2851);
```

Comment:
lol I meant the whole display value to ensure the whole thing matched what's expected (including format string) but can also revert to the last commit for these if you'd prefer that :)

Reply:
will do (revert that is) :)